eslint.config.mjs

```diff
@@ -0,0 +1,366 @@
+import jest from 'eslint-plugin-jest';
+import lwcInternal from '@lwc/eslint-plugin-lwc-internal';
+import _import from 'eslint-plugin-import';
+import header from 'eslint-plugin-header';
+import { fixupPluginRules } from '@eslint/compat';
+import globals from 'globals';
+import js from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default [
+    {
+        ignores: [
+            '**/node_modules/',
+            '**/dist/',
+            '**/lib/',
```

Comment:
```suggestion
```
We don't have any `lib` directories to ignore!

Reply:
I remember getting these values from the `.eslintignore` file.